src/screens/NotificationScreen/components/EventNotification/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import dayjs from 'dayjs';
+import { styles } from './style';
+import { dateFormat } from '../../../../components/constants';
+import { THEME_COLORS } from '../../../../components/theme';
+import { mockEventInfo, mockTeamInfo } from '../../../../shared/zustand/event/mockData';
+    
+const EventNotification = ({ event, eventType, onPress }) => {
+  return (
+    <TouchableOpacity style={styles.eventItem} onPress={() => {
+      const eventId = event['eventId'];
```

Comment:
Try accessing the `eventId` attribute using the dot notation for consistency. i.e., `event.eventId`